chef/cookbooks/ceph/recipes/radosgw.rb

```diff
@@ -61,6 +61,27 @@
   include_recipe "ceph::radosgw_keystone"
 end
 
+# Create admin user and add caps to access buckets and users
+ruby_block "create radosgw-admin-access" do
+  block do
+    uid = "rgw_admin"
+
+    Chef::Log.info("Creating RadosGW admin user with capabilities")
+    user_add = Mixlib::ShellOut.new("radosgw-admin user create --display-name=RadosGW_Admin --uid=#{uid}")
+    user_add.run_command
+    user_add.error!
+
+    caps_bucket = Mixlib::ShellOut.new("radosgw-admin caps add --uid=#{uid} --caps='buckets=*'")
+    caps_bucket.run_command
+    caps_bucket.error!
+
+    caps_users = Mixlib::ShellOut.new("radosgw-admin caps add --uid=#{uid} --caps='users=*'")
+    caps_users.run_command
+    caps_users.error!
+  end
+  action :create
```

Comment:
isn't there some only_if / not_if missing? otherwise this causes an error on the 2nd run, doesn't it?